README.md

```diff
@@ -1,2 +1,31 @@
 # team 2 LMNH group project
-This is a repository for tracking the LMNH project to protect the precious plants
+This is the repository for a project with the Liverpool Natural History Museum. The museum hosts a large conservatory that houses a large variety of plants from around the world. We have created an ETL pipeline to aid in monitoring the sheer number of plants to ensure that they are being kept in their appropriate environments.
+
+
+## Pipeline architecture
+This repository contains all the files needed to set up our ETL (extract, transform and load) pipeline. Our Python ETL scripts are dockerised into a container that is then implemented into our cloud infrastructure:
```

Comment:
You are missing an Oxford comma in the list.

Reply:
(extract, transform and load) > (extract, transform, and load)

It sort of matters because they are separate tasks.

e.g.
“I owe my success to the inspiration of my parents, the Virgin Mary and God.”